feature/metis/conversation/src/test/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ui/reply/ReplyTextFieldVisibilityUITest.kt

```diff
@@ -0,0 +1,84 @@
+package de.tum.informatics.www1.artemis.native_app.feature.metis.conversation.ui.reply
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import de.tum.informatics.www1.artemis.native_app.core.common.test.EndToEndTest
+import de.tum.informatics.www1.artemis.native_app.feature.metis.conversation.BaseThreadUITest
+import de.tum.informatics.www1.artemis.native_app.feature.metis.conversation.TestInitialReplyTextProvider
+import de.tum.informatics.www1.artemis.native_app.feature.metis.conversation.service.impl.EmojiServiceStub
+import de.tum.informatics.www1.artemis.native_app.feature.metis.conversation.ui.chatlist.ChatListItem
+import de.tum.informatics.www1.artemis.native_app.feature.metis.conversation.ui.chatlist.MetisChatList
+import de.tum.informatics.www1.artemis.native_app.feature.metis.conversation.ui.chatlist.PostsDataState
+import de.tum.informatics.www1.artemis.native_app.feature.metis.shared.db.pojo.PostPojo
+import junit.framework.TestCase.assertTrue
+import kotlinx.coroutines.CompletableDeferred
+import org.junit.Test
+import org.junit.experimental.categories.Category
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@Category(EndToEndTest::class)
+@RunWith(RobolectricTestRunner::class)
+class ReplyTextFieldVisibilityUITest : BaseThreadUITest() {
+
+    @Test
+    fun `test GIVEN the thread view is shown containing one post and three answer posts WHEN the markdown text field is clicked THEN the keyboard is shown below the markdown text field`() {
+        setupThreadUi(posts[0]) { CompletableDeferred() }
+        runTest()
+    }
+
+    @Test
+    fun `test GIVEN the chat list containing three posts is shown WHEN the markdown text field is clicked THEN the keyboard is shown below the markdown text field`() {
+        setupChatUi(posts)
+        runTest()
+    }
```

Comment:
Tbh I am not sure whether this is possible with our current UI test setup, but can you look into whether we could also add another test where `enableEdgeToEdge()` is set for the tests? 
Because I checked whether your tests would have been able to catch the problem initially introduced by `enableEdgeToEdge` where the keyboard covered the textField. I reverted the padding and inset changes in `ConversationChatListScreen`, but the tests still pass

Reply:
That's indeed problematic. I did some research on this, but apparently, there is no way to simulate `enableEdgeToEdge()` in compose UI tests.

Reply:
I created this issue #121 to make sure we don't forget about it and maybe come up with an alternative test setting.